source/stylesheets/header.scss

```diff
@@ -0,0 +1,54 @@
+details > summary {
+	padding: 1.05em;
+	color: white;
+	font-size: 1.8em;
+}
+
+details {
+	background: #ed3e44;
+	box-shadow: 0.5px 0.5px 25px 0.5px grey;  
+}
+
+details[open] {
+	box-shadow: 0.5px 0.5px 25px 0.5px grey;  
```

Comment:
Is this duplicate rule necessary? 

Also, What's the reasoning behind using `0.5px`?
 
+1 for using British English spelling of `grey`